rbac/server/swagger/index.html

```diff
@@ -1999,7 +1999,74 @@
       							}
       						}
       					}
-      				}
+      				},
+							"delete": {
```

Comment:
After viewing the swagger ui at localhost:8000/api/swagger, it appears there are some formatting errors with the DELETE /users/{id} section. For example, there's an undefined parameter that I can pass into the API call. Also, the responses seem to be missing as well.